spec/converterbase_spec.rb

```diff
@@ -293,6 +293,105 @@ def to_ruby(match, m1, m2, open_close)
                "追撃をしない俺に振り返り、｜再度《パワースラッシュ 》"
       end
     end
+
+    describe "｜を使ったルビ位置の指定" do
+      it "《 》基本" do
+        expect(to_ruby( \
+               "｜山田太郎《やまだたろう》",
+               "｜山田太郎", "やまだたろう", ["《", "》"])).to eq \
+               "｜山田太郎《やまだたろう》"
+      end
+      it "（ ）基本" do
+        expect(to_ruby( \
+               "｜山田太郎（やまだたろう）",
+               "｜山田太郎", "やまだたろう", ["（", "）"])).to eq \
+               "｜山田太郎《やまだたろう》"
+      end
+      it "《 》ルビを振りたくない場合" do
+        expect(to_ruby( \
+               "山田太郎｜《やまだたろう》",
+               "山田太郎｜", "やまだたろう", ["《", "》"])).to eq \
+               "山田太郎《やまだたろう》"
+      end
+      it "（ ）ルビを振りたくない場合" do
+        expect(to_ruby( \
+               "山田太郎｜（やまだたろう）",
+               "山田太郎｜", "やまだたろう", ["（", "）"])).to eq \
+               "山田太郎（やまだたろう）"
+      end
+    end
+
+    describe "ルビの自動対応" do
+      it "（ ）自動" do
+        expect(to_ruby( \
+               "HP（ヒットポイント）",
+               "HP", "ヒットポイント", ["（", "）"])).to eq \
+               "｜HP《ヒットポイント》"
+      end
+      it "《 》自動" do
+        expect(to_ruby( \
+               "ＭＰ《マジックポイント》",
+               "ＭＰ", "マジックポイント", ["《", "》"])).to eq \
```

Comment:
__[rubocop]__
* Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.

<!-- Comment by [SideCI](https://sideci.com) -->